### PR TITLE
arrow: Add Xuwei to ccs

### PR DESCRIPTION
**Pull request by mapleFU**
Disclaimer: I'm a arrow committer and would working on the fuzz testing

### CODE REVIEW

**Comment by mapleFU** [comment]
cc @pitrou

File: projects/arrow/project.yaml [hunk 1]
```diff
@@ -14,4 +14,5 @@ auto_ccs:
   - "wesmckinn@gmail.com"
   - "xhochy@gmail.com"
   - "will.jones127@gmail.com"
+  - "maplewish117@gmail.com"
```
Comment (pitrou) [inline projects/arrow/project.yaml:17]: Can you keep this list in alphabetical order?

**Review by pitrou — CHANGES_REQUESTED** [review]

**Comment by pitrou** [comment]
Also cc @emkornfield @felipecrv

File: projects/arrow/project.yaml (hunk 1 above)
Comment (mapleFU) [reply projects/arrow/project.yaml]: done

**Review by felipecrv — APPROVED** [review]

**Review by emkornfield — APPROVED** [review]

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**